app/main.py

```diff
@@ -1,6 +1,22 @@
 from typing import Callable
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    @wraps(func)
+    def inner(*args) -> None:
```

Comment:
`inner` does not return `None`, it could return `Any`. Change your return type annotation.